snap/snapcraft.yaml

```diff
@@ -0,0 +1,94 @@
+name: jitsi
```

Comment:
Can we call this whole thing Jitsi Meet? Jitsi was the legacy desktop client, which can be a ssource of confusion.

Reply:
You can use whatever is most suitable for the name and description.